core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/CompactEntityModel.scala

```diff
@@ -63,7 +63,8 @@ case class KeysRecord(id: Long, workspaceId: UUID, entityType: String, attribute
 case class RefPointerRecord(fromId: Long, toId: Long)
 
 /** all reference pointers from one entity to all its reference targets */
-case class RefPointers(fromId: Long, toIds: Set[Long])
+// TODO CORE-497: rename
```

Comment:
rename the class?

Reply:
Would you mind if I made a follow-on PR to handle this rename and the other TODOs in CompactEntityProvider which would change the return value of `findAllReferences`? I'd like to get this schema change in as soon as possible (assuming approval) to unblock the other tickets.

Reply:
Yeah no problem, just wasn't sure the intention of this TODO

Reply:
ah yeah - I find the name `RefPointers` awkward, especially because we often work with `Set[RefPointers]`, leading to code like:
```scala
val refPointerses: Set[RefPointers] = …
```